### PR TITLE
Use marketplace prefix in plugin name

### PR DESCRIPTION
**Pull request by flagbird**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,16 +20,16 @@
...
         }
     },
     "autoload-dev": {
         "psr-4": {
-            "EinsUndEins\\PluginTransactionMailExtender\\Tests\\": "tests/"
+            "EinsUndEins\\SmartInboxConnector\\Tests\\": "tests/"
         }
     },
     "extra": {
-        "shopware-plugin-class": "EinsUndEins\\PluginTransactionMailExtender\\TransactionMailExtender",
```
Comment (Vogaeael) [inline composer.json:32]: @flagbird the namespace is not correct

**Review by Vogaeael — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (flagbird) [reply composer.json:32]: Good catch, even though it confused me for a minute by having a comment on the wrong side of the diff. :sweat_smile: 

Change is done.

File: composer.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
     "description": "Extends Shopware 6 transaction mails with a Schema.org conform HTML content",
     "type": "shopware-platform-plugin",
     "license": "MIT",
+    "version": "1.0.1",
```
Comment (flagbird) [inline composer.json:6]: I'm not sure about the version. In theory it's a breaking change, but afaik no one uses this plugin at the moment.

**Review by robin-homberg — APPROVED** [review]
lgtm

**Pull request merged**